backend/models/postgis/project.py

```diff
@@ -723,6 +723,14 @@ def get_project_stats(self) -> ProjectStatsDTO:
                     )
                     project_stats.average_validation_time = average_validation_time
 
+        time_to_finish_mapping = (
```

Comment:
@JorgeMartinezG - why do use this over `total_mapping_seconds`?

Reply:
Understand this is for time estimates on remaining tasks - in that case, how do we handle projects with 0 tasks mapped/validated